SampleProject/VOKCoreDataManagerTests/ManagedObjectImportExportTests.m

```diff
@@ -452,7 +452,8 @@ - (void)testFetchWithURI
     person = nil;
     [[[VOKCoreDataManager sharedInstance] managedObjectContext] reset];
 
-    VIPerson *personFromURI = [[VOKCoreDataManager sharedInstance] existingObjectAtURI:uri forManagedObjectContext:nil];
+    VIPerson *personFromURI = (VIPerson *)[[VOKCoreDataManager sharedInstance] existingObjectAtURI:uri
```

Comment:
It needs the cast on Xcode < 7 to avoid a warning.  Without the `__kindof` and with my changes, this method now returns a `NSManagedObject *` instead of an `id`.